static/js/controllers/contacts-edit.js

```diff
@@ -186,8 +134,11 @@ var _ = require('underscore');
           });
       };
 
-      setupSchemas()
-        .then(getContact)
+      $scope.unmodifiedSchema = ContactSchema.get();
+      $scope.dependentPersonSchema = ContactSchema.get('person');
+      delete $scope.dependentPersonSchema.fields.parent;
```

Comment:
Does that not delete it off the ContactSchema?

(I see this was in the original code. Still, seems wrong?)